uvobot.rb

```diff
@@ -1,13 +1,20 @@
 require 'dotenv'
 require 'date'
-require './lib/uvobot'
-require './lib/slack_notifier'
-require './lib/uvo_scraper'
-require './lib/uvo_parser'
+require_relative 'lib/uvobot'
 
 Dotenv.load
+discourse_client = Uvobot::DiscourseClient.new(
+  ENV.fetch('DISCOURSE_URL'),
+  ENV.fetch('DISCOURSE_API_KEY'),
+  ENV.fetch('DISCOURSE_USER')
```

Comment:
Je to v pohode, kriticke bude cim skor #3 a #8. Lebo tie oznamenia co teraz tahame sa niekedy viazu k jednemu obstaravaniu. vyhlasenie sutaze, oznamenie o vysledku a kopec dalsich veci. Chceli by sme to drzat v jednom topicu pre kazde obstarko, aby to nespamovalo forum.